README.md

```diff
@@ -1,2 +1,2 @@
 # test
-:cop: `move along nothing to see here`
+:rainbow: amazing things to see here!!!
```

Comment:
are three !'s really appropriate here? it should either be two for consistency with the two :'s or seven for the different colours of the rainbow

Reply:
`:rainbow: amazing things to see here!!!`
is equivalent to:
`rainbow amazing things to see here!`

Reply:
:shipit: